src/lib/es5.d.ts

```diff
@@ -505,6 +505,10 @@ interface TemplateStringsArray extends ReadonlyArray<string> {
     readonly raw: ReadonlyArray<string>;
 }
 
+interface ImportMeta {
+    [propertyName: string]: any;

```

Comment:
There is no way to opt out of this one... users who want strict definition can not undo the index signature. consider not adding it and letting say node.d.ts add it.